moveos/moveos/src/moveos.rs

```diff
@@ -190,6 +193,11 @@ impl MoveOS {
                 let args = session
                     .resolve_args(&tx_context, loaded_function, function.args)
                     .map_err(|e| e.finish(Location::Undefined))?;
+
+                let func =
+                    session.load_function(&function.function_id, function.ty_args.as_slice())?;
+                moveos_verifier::verifier::verify_entry_function(func, session.runtime_session())?;
+
```

Comment:
`verify_entry_function` should be before the `resolve_args`.

Reply:
Got it. I will submit a pull request to modify it later.